<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenRcon</class>
 <widget class="QMainWindow" name="OpenRcon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionReport_bug"/>
    <addaction name="actionDocumentation"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuToolbars">
     <property name="title">
      <string>Toolbars</string>
     </property>
     <addaction name="actionConnection"/>
     <addaction name="actionQuickconnect"/>
    </widget>
    <addaction name="actionHome"/>
    <addaction name="menuToolbars"/>
   </widget>
   <widget class="QMenu" name="menuApplication">
    <property name="title">
     <string>Application</string>
    </property>
    <addaction name="actionServermanager"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuApplication"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar_sm">
   <property name="windowTitle">
    <string>Servermanager</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionServermanager"/>
  </widget>
  <widget class="QToolBar" name="toolBar_qc">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Quickconnect</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Connection</string>
   </property>
  </action>
  <action name="actionQuickconnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Quickconnect</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset resource="data/graphics/graphics.qrc">
     <normaloff>:/data/graphics/icons/go-home.png</normaloff>:/data/graphics/icons/go-home.png</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Home</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="data/graphics/graphics.qrc">
     <normaloff>:/data/graphics/icons/preferences-system.png</normaloff>:/data/graphics/icons/preferences-system.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="icon">
    <iconset resource="data/graphics/graphics.qrc">
     <normaloff>:/data/graphics/icons/documentation.png</normaloff>:/data/graphics/icons/documentation.png</iconset>
   </property>
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionReport_bug">
   <property name="icon">
    <iconset resource="data/graphics/graphics.qrc">
     <normaloff>:/data/graphics/icons/tools-report-bug.png</normaloff>:/data/graphics/icons/tools-report-bug.png</iconset>
   </property>
   <property name="text">
    <string>Report bug</string>
   </property>
  </action>
  <action name="actionServermanager">
   <property name="icon">
    <iconset resource="data/graphics/graphics.qrc">
     <normaloff>:/data/graphics/icons/network-server.png</normaloff>:/data/graphics/icons/network-server.png</iconset>
   </property>
   <property name="text">
    <string>Servermanager</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionPluginmanager">
   <property name="text">
    <string>Pluginmanager</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="data/graphics/graphics.qrc">
     <normaloff>:/data/graphics/openrcon.png</normaloff>:/data/graphics/openrcon.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="data/graphics/graphics.qrc"/>
 </resources>
 <connections/>
</ui>
